README.md

```diff
@@ -342,43 +289,59 @@ bq extract --destination_format=NEWLINE_DELIMITED_JSON
 The list of projects that are checked is available in the
 [`cron/data/projects.csv`](https://github.com/ossf/scorecard/blob/main/cron/data/projects.csv)
 file in this repository. If you would like us to track more, please feel free to
-send a Pull Request with others.
-
-**NOTE**: Currently, these lists are derived from **projects hosted on GitHub
+send a Pull Request with others. Currently, this list is derived from **projects hosted on GitHub
 ONLY**. We do plan to expand them in near future to account for projects hosted
 on other source control systems.
 
-## Adding a Scorecard Check
+**NOTE**: The public dataset includes data from earlier version of Scorecards that used Pass/Fail scoring systems. Previously, each check returned a **Pass / Fail** decision, as well as a confidence score
```

Comment:
The public data is still using Pass/Fail format. We haven't yet exposed the 0-10 scores in our BQ dataset.

Reply:
Tweaked to convey that scoring is still pass/fail for BQ datasets.